index.html

```diff
@@ -58,6 +58,237 @@ <h2 class="connect"> LET’S CONNECT </h2>
                     </a>
                 </li>
             </ul>
-        </section>            
+        </section>
+        <section class="works">
+            <div class="works-container">
+                <div class="item">
+                    <div class="card">
+                        <div class="card-body">
+                            <img class="card-img" src="images/Snapshoot.png" alt="project one">
+                            <h3 class="card-title"> Tonic </h3>
+                            <div class="frame2">
+                                <div class="client">
+                                    <h3> CANOPY </h3>
+                                </div>
+                                <div>
+                                    <ul class="roles">
+                                        <li class="role"> Back End Dev </li>
+                                        <li class="role"> 2015 </li>
+                                    </ul>
+                                </div>
+                            </div>
+                            <div>
+                                <p class="description">
+                                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                </p>
+                            </div>
+                            <div>
+                                <ul class="tags">
+                                    <li class="tag">
+                                        <a href="#">html</a>
+                                    </li>
+                                    <li class="tag">
+                                        <a href="#">css</a>
+                                    </li>
+                                    <li class="tag">
+                                        <a href="#">javaScript</a>
+                                    </li>
+                                </ul>
+                            </div>
+                            <button class="btn btn-enable"> See Project </button>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [ OTIONAL] You really did a very great thing in this project, just I need to give a hint for this one. just everything you need to list something in your development, it's much better to use `ul` tag just to keep using semantic Html :smiley: